[NET-5327] Templated policies api/cli docs (#19270)
hashicorp/consul: website/content/api-docs/acl/templated-policies.mdx

}
```

## Preview a Templated Policy

This endpoint shows the resulting policy created from a templated policy. Useful to understand what templated variables are required/missing from your request.

| Method | Path          | Produces           |
| ------ | ------------- | ------------------ |
| `POST`  | `/acl/templated-policy/preview/:name` | `application/json` |

The table below shows this endpoint's support for
[blocking queries](/consul/api-docs/features/blocking),
[consistency modes](/consul/api-docs/features/consistency),
[agent caching](/consul/api-docs/features/caching), and
[required ACLs](/consul/api-docs/api-structure#authentication).

| Blocking Queries | Consistency Modes | Agent Caching | ACL Required |
| ---------------- | ----------------- | ------------- | ------------ |
| `NO`             | `none`            | `none`        | `acl:read`  |

The corresponding CLI command is [`consul acl templated-policy preview`](/consul/commands/acl/templated-policy/preview).

### Path Parameters

- `name` `(string: <required>)` - Specifies the name of the ACL templated policy to preview.

### Query Parameters

> - `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace of the acl templated policy is previewed in.

Author: templated policies are available throughout all namespaces: so more like: Specifies the namespace of the acl templated policy will be used

### Query parameters

> - `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace of the acl templated policy will be used.
>   You can also [specify the namespace through other methods](#methods-to-specify-namespace).

Reviewer: ```suggestion
- `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace that the policy created from the template applies to.
  You can also [specify the namespace through other methods](#methods-to-specify-namespace).
```
Is that correct? Or are you specifying the namespace where Consul retrieves the specified template?
